src/test/java/Tests/object/BCommentTable.java

```diff
@@ -62,6 +74,22 @@ public boolean isCommentInactive(String state) {
         }
     }
 
-}
-
+    @Step("Verify presence of the category in comment categories ")
+    public boolean verifyCategories(String expected) throws NoSuchElementException {
+        try {
+            List<WebElement> categories = this.driver.findElements(By.xpath("//*[@class='categorycolumn']"));
+            for (WebElement row : categories) {
+                int i = 0;
+                row = categories.get(i);
+                String actual = row.getText();
+                System.out.println(row.getText());
+                if (actual.contains(expected))
```

Comment:
It will return `true` if at least one line has expected category. And as I understand this method, it has to return `false` if even one line doesn't have the expected category.

Reply:
fixed